lib/app_funcs.sh

```diff
@@ -16,7 +16,7 @@ function copy_hex() {
   mkdir -p ${build_path}/.hex
 
   # hex file names after elixir-1.1 in the hex-<version>.ez form
-  full_hex_file_path=$(ls -t ${HOME}/.mix/archives/hex-*.ez | head -n 1)
```

Comment:
my bash isn't that 1337. what are you changing exaclty? and does this break backwards compat?